rustfmt.toml

```diff
@@ -1 +1,3 @@
+match_block_trailing_comma = true
 newline_style = "Native"
+where_trailing_comma = true
```

Comment:
I am a bit wary about the trailing comma on match arms. When writing code I omit it but since rustfmt takes care of adding it anyway I guess I don't really mind it. It does make things more consistent.